impl/client/src/main/javascript/web/util/_a11y.js

```diff
@@ -35,6 +35,358 @@ define(["./_focus"], function(focusUtil) {
     delete: 46
   };
 
+  /*
+   * This software or document includes material copied from or derived from https://www.w3.org/WAI/content-assets/wai-aria-practices/patterns/listbox/examples/js/listbox.js
+   * Copyright © 2023 World Wide Web Consortium. https://www.w3.org/Consortium/Legal/2023/doc-license
+   *
+   */
+  function makeAccessibleListbox(elem) {
```

Comment:
Unlike the other utilities provided by this file, this one is not returning a "disposable" handle which allows the caller to cleanup the attached event listeners.

Additionally, as seen in the associated Analyzer code, some non-trivial operations are being done externally (e.g. clearing the selected options, clearing the focused option, getting the list of selected options). This could/should be provided as methods.

One way to achieve both of the above requirements would be to return an object that would expose such methods (e.g. `clearSelectedOptions()`, `clearActiveOption()`, `getSelectedOptions()`, `destroy()`).